ParkingGarageRentals/Program.cs

```diff
@@ -32,12 +32,22 @@ static void Main(string[] args)
                 Console.WriteLine("DailyPass: $5 (1), WeeklyPass: $15 (2), MonthlyPass: $30 (3), YearlyPass: $90 (4)");
                 Console.Write("Please choose a parking pass (Enter number 1 - 4): ");
 
-                passNumberSelected = Convert.ToInt32(Console.ReadLine());
+                try
+                {
+                    passNumberSelected = Convert.ToInt32(Console.ReadLine());
+                }
+                catch (FormatException)
+                {
+                    Console.WriteLine("Must enter a valid number.");
+                }
```

Comment:
- [ ] do-while can be used here to ensure the user enters a valid number before continuing to logic below
- [ ] a sentinel value can also be used to exit the menu